lib/addons/flagger/my_new_addon.ts

```diff
@@ -0,0 +1,65 @@
+// filename=lib/kubevious_addon.ts
+import 'source-map-support/register';
+import * as blueprints from '../../../lib';
+import { Construct } from 'constructs';
+import { setPath } from '../../utils';
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {
+  version?: string,
+  ingressEnabled?: boolean,
+  flaggerServiceType?: string,
```

Comment:
if it allows only specific values, it should be an enum

Reply:
Example from outline I did not remove yet. If I need these later I'll keep that in mind for the design.